fastplotlib/layouts.py

```diff
@@ -111,9 +119,17 @@ def __init__(
                 renderer=renderer
             )
 
+            self.indexers[i, j] = GridPlotIndexer((i,j))
+
         self._animate_funcs: List[callable] = list()
         self._current_iter = None
 
+    def __getitem__(self, index: tuple):
+        return self.indexers[index[0], index[1]].name
+
+    def __setitem__(self, index: tuple, name: str):
```

Comment:
this is not necessary, the indexer should be read-only. Since `__getitem__()` returns an instance of `GridPlotIndex`, it can be named right there.

example:
```python
indexer = gp[i, j] # returns `GridPlotIndexerInstance`
indexer.name = "bah"

# can also just directly do
gp[i, j].name = "bah"
```